packages/core/src/util/spanFilter.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ */
+
+'use strict';
+
+// List of span types to allowed to ignore
+const IGNORABLE_SPAN_TYPES = ['redis'];
+
+/**
+ * @param {import('../core').InstanaBaseSpan} span
+ * @param {import('../tracing').IgnoreEndpoints} endpoints
+ * @returns {boolean}
+ */
+function shouldIgnore(span, endpoints) {
+  // Skip if the span type is not in the ignored list
+  if (!IGNORABLE_SPAN_TYPES.includes(span.n)) {
+    return false;
+  }
+  const operation = span.data?.[span.n]?.operation;
+
+  if (operation && endpoints[span.n]) {
+    return endpoints[span.n].includes(operation);
```

Comment:
issue: `includes` does substring matches. I'd suggest to use exact match.

Reply:
updated code